src/modules/layoutUiRules/rules.js

```diff
@@ -79,8 +95,8 @@ const pivotTableRules = {
 }
 
 const visTypeToRules = {
-    [VIS_TYPE_COLUMN]: defaultRules,
-    [VIS_TYPE_STACKED_COLUMN]: defaultRules,
+    [VIS_TYPE_COLUMN]: dualCategoryRules,
```

Comment:
Breaking change? @janhenrikoverland

Reply:
With this update, the current version of DV will be able to add 2 items to Category but won’t be able to process the implications properly.
![image](https://user-images.githubusercontent.com/12590483/88933037-2912d100-d27f-11ea-958a-7f104e2b240b.png)

Reply:
Turns out there was an issue with the backend (debug/dev). Testing @dhis2/analytics@8.2.0-alpha.1 in DV master against debug/2.34dev works! So seems like there's no breaking changes.

![image](https://user-images.githubusercontent.com/12590483/89010751-79d20a80-d30f-11ea-95ba-32843b4d175f.png)